internal/cmd/list.go

```diff
@@ -35,6 +36,10 @@ func (c *List) Execute(opts *commander.CommandHelper) {
 		os.Exit(1)
 	}
 
+	sort.Slice(namespace.Accounts, func(i, j int) bool {
```

Comment:
Sorting before save (on add/update/delete) would be more efficient, but does not add much cost here unless someone has 2million accounts in one namespace (they shouldn't), so fine by me.